modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -54,6 +54,51 @@
 public class RelatedObjectEntryResourceImpl
 	extends BaseRelatedObjectEntryResourceImpl {
 
+	@Override
+	public void deleteObjectRelationshipMappingTableValues(
+			String previousPath, Long objectEntryId,
+			String objectRelationshipName, Long relatedObjectEntryId)
+		throws Exception {
+
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153324"))) {
+			throw new NotFoundException();
+		}
+
+		ObjectDefinition systemObjectDefinition = _getSystemObjectDefinition(
+			previousPath);
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.
+				getObjectRelationshipByObjectDefinitionId(
+					systemObjectDefinition.getObjectDefinitionId(),
+					objectRelationshipName);
+
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.getObjectDefinition(
```

Comment:
I think this invocation is weird. We use the `objectDefinition` element just to check in the `_getPrimaryKey1` method if its identifier is the same id than the `systemObjectDefinition` identifier, but you can know that just by comparing the `objectRelationship.getObjectDefinitionId1()` value with the `systemObjectDefinition` identifier, not necessary to get the `objectDefinition` element.

It happens the same in the put method